salt/modules/win_iis.py

```diff
@@ -0,0 +1,164 @@
+# -*- coding: utf-8 -*-
+'''
+Microsoft IIS site management via WebAdministration powershell module
+
+:platform:      Windows
+
+'''
+
+
+from __future__ import absolute_import
+
+# Import salt libs
+import salt.utils
+
+# Define the module's virtual name
+__virtualname__ = 'win_iis'
+
+
+def __virtual__():
+    '''
+    Load only on Windows
+    '''
+    if salt.utils.is_windows():
+        return __virtualname__
+    return False
+
+
+def _srvmgr(func):
+    '''
+    Execute a function from the WebAdministration PS module
+    '''
+
+    return __salt__['cmd.run'](
+        'Import-Module WebAdministration; {0}'.format(func),
```

Comment:
Got it.

I think the current code should suffice since I'm not directly putting content into the private method for the shell command. The commands need to be parsed in other methods that actually build out the command to send to the server. If you don't agree I can continue to search for another solution.